sdk/clientcore/core/src/main/java/io/clientcore/core/implementation/http/client/JdkHttpClient.java

```diff
@@ -44,4 +45,12 @@ public JdkHttpClient(Object httpClient, Set<String> restrictedHeaders, Duration
     public Response<BinaryData> send(HttpRequest request) {
         throw LOGGER.throwableAtError().log(ERROR_MESSAGE, UnsupportedOperationException::new);
     }
+
+    @Override
+    public CompletableFuture<Response<BinaryData>> sendAsync(HttpRequest request) {
+        CompletableFuture<Response<BinaryData>> failedFuture = new CompletableFuture<>();
+        failedFuture
+            .completeExceptionally(LOGGER.throwableAtError().log(ERROR_MESSAGE, UnsupportedOperationException::new));
+        return failedFuture;
+    }
```

Comment:
I'm confused by this class. The JavaDoc references 'DefaultHttpClient', the type is 'JdkHttpClient', and the send / sendAsync impls are unsupported? Why does this class exist?